secret-recipes/src/actions/index.js

```diff
@@ -7,14 +8,15 @@ export const FETCHING_RECIPES_FAIL = "FETCHING_RECIPES_FAIL"
 export const getRecipes = token => dispatch => {
   // localStorage.setItem('token', token)
   dispatch({ type: FETCHING_RECIPES_START })
-  axios.get('https://lambdaschool-cookbook2.herokuapp.com/recipes', {
-    headers: {
-      Authorization: token
+  axiosWithAuth()
+    .get('https://lambdaschool-cookbook2.herokuapp.com/recipes', {
+      headers: {
+        Authorization: token
+      }
     }
-  }
-  )
+    )
     .then(res => {
-      console.log(res)
+      console.log("something", res)
```

Comment:
Don't forget to remove unneeded console.logs once everything is working. It keeps the code clean